app/presenters/webauthn_setup_presenter.rb

```diff
@@ -24,11 +25,18 @@ def initialize(
     @url_options = url_options
   end
 
-  def image_path
-    if @platform_authenticator
-      'platform-authenticator.svg'
-    else
-      'security-key.svg'
+  def learn_more_html
```

Comment:
Might be good to have some test coverage for this method in `webauthn_setup_presenter_spec.rb`